packages/cli/src/modules/breaking-changes/rules/v2/removed-nodes.rule.ts

```diff
@@ -0,0 +1,60 @@
+import { Service } from '@n8n/di';
+
+import type { DetectionResult, BreakingChangeMetadata, CommonDetectionInput } from '../../types';
+import { BreakingChangeSeverity, BreakingChangeCategory, IssueLevel } from '../../types';
+import { AbstractBreakingChangeRule } from '../abstract-rule';
+
+@Service()
+export class RemovedNodesRule extends AbstractBreakingChangeRule {
+	private readonly REMOVED_NODES = [
+		'n8n-nodes-base.spontit',
+		'n8n-nodes-base.crowdDev',
+		'n8n-nodes-base.kitemaker',
+	];
+
+	getMetadata(): BreakingChangeMetadata {
+		return {
+			id: 'removed-nodes-v2',
+			version: 'v2',
+			title: 'Removed Deprecated Nodes',
+			description: 'Several deprecated nodes have been removed and will no longer work',
+			category: BreakingChangeCategory.WORKFLOW,
+			severity: BreakingChangeSeverity.CRITICAL,
+		};
+	}
+
+	async detect({ workflows }: CommonDetectionInput): Promise<DetectionResult> {
+		const result = this.createEmptyResult();
+
+		try {
+			for (const workflow of workflows) {
+				const removedNodes = workflow.nodes.filter((n) => this.REMOVED_NODES.includes(n.type));
+				if (removedNodes.length === 0) continue;
+
+				result.affectedWorkflows.push({
+					id: workflow.id,
+					name: workflow.name,
+					active: workflow.active,
+					issues: removedNodes.map((node) => ({
+						title: `Node '${node.type}' has been removed`,
```

Comment:
If there are multiple of these nodes in the same WF, the same issue will be shown multiple times. We could show the node names